### DISCUSSION TITLE
Application logger: server-time versus local time

### DISCUSSION THREAD

**Discussion by elkede** [pimcore/pimcore#16984] [General]
When you look at the application logger: the timestamp of the logs is the server-time. It is a bit weird to see messages from the future but it is explainable :)
However, when you want to filter the logs, the local time is used...

![image](https://github.com/pimcore/pimcore/assets/165149881/a3bdc21d-13a5-48a3-84b1-ce4330905ddf)

**Comment by blankse** [comment]
@elkede Which version do you use? There was a bug in Pimcore 11.0.0 - 11.1.6
https://github.com/pimcore/pimcore/pull/16678

The timestamp should be saved in UTC and the admin ui should show it in the user timezone.
